### PR TITLE
Redesign HomePage and Navbar

### PR DESCRIPTION
**Pull request by ttmtu2003**
**Input & Output:**
Input: 
Screen Shot 2022-01-30 at 12 48 03 AM

Output: updated on web
Screen Shot 2022-01-30 at 3 50 33 PM

Fixed:
Screen Shot 2022-01-24 at 9 25 35 PM
Screen Shot 2022-01-24 at 9 25 46 PM

Screen Shot 2022-03-07 at 11 32 32 AM

Process:

Change layout structure of UserNavbar components
Change design of sign in/sign up buttons
Change color, font style
Add picture frame in hero section
Attach link to SCE instagram

### CODE REVIEW

File: src/Components/Navbar/UserNavbar.js [hunk 1]
```diff
@@ -76,23 +43,24 @@ export default function UserNavBar(props) {
       ].join('')
     }
   ];
-  const unauthedRoutes = [{ title: 'Events', route: '/events' },
-    { title: 'Discord Bot', route: '/discordSJSU' }];
+  const unauthedRoutes = [{ title: 'About', route: '/about' },
```
Comment (evanugarte) [inline src/Components/Navbar/UserNavbar.js:46]: part of suggestion 2

```suggestion
  const unauthedRoutes = [{ title: 'About', route: '/#about' },
```

then go into [`src/Components/Jumbotron/Jumbotron.js:21`](https://github.com/SCE-Development/Core-v4/blob/075e17067d90a8170da4e4b3ea3c54b4a92818b9/src/Components/Jumbotron/Jumbotron.js#L21) and do:

```html
<div id="about">
```

now, the link will take the user to the about section on the homepage.

**Review by evanugarte — CHANGES_REQUESTED** [review]
Looks great! Couple things.

1. Slightly change the logic in the css to show the hamburger menu earlier, there is some overlap:
image

2. The "About" page doesn't exist, but we have an "About" section on the homepage, you can update the link to point there? See associated comment i made for how to do this

3. The "Services" page doesn't exist. are we going to make a page for this?

File: src/Components/Navbar/AdminNavbar.js [hunk 2]
```diff
@@ -33,7 +33,7 @@ export default function AdminNavbar(props) {
         <NavbarBrand href='/'>Admin Dashboard
         </NavbarBrand>
         <div id='admin-toggler'>
-          <DarkMode />
+          {/* <DarkMode /> */}
```
Comment (evanugarte) [inline src/Components/Navbar/AdminNavbar.js:36]: why is this commented out?

Comment (ttmtu2003) [reply src/Components/Navbar/AdminNavbar.js:36]: there's no dark mode on the new design :/

**Review by andrewtedi — CHANGES_REQUESTED** [review]
Really like the design and implementation!

The dropdown menu shows the Sign Up and Sign In items when the screen becomes smaller, however, the Navbar does not display these links. There should also be a slight tweaking of spacing between the Sign Up and Sign In items as well for consistency.
dropdown

**Review by allen-tran — COMMENTED** [review]
This is so sick. Agree with Andrew on the suggestions, should be even more awesome after those changes :)

**Review by allen-tran — APPROVED** [review]
This is a really thicc pr, great job!

**Review by neelp03 — CHANGES_REQUESTED** [review]
New UI is 🔥 
One thing I noticed was that the components on the register page were misaligned: 
image

**Review by andrewtedi — CHANGES_REQUESTED** [review]
image
Looks good! Just maybe one small change. The Dark mode toggle should be a slider instead of just a check box. I'd look into how Dark Mode is implemented and the CSS surrounding it.

**Review by evanugarte — APPROVED** [review]
This pr is removing over 20MB of unused images from Core-v4!! This is huge!! Thank you so much Tu!!

**Review by neelp03 — APPROVED** [review]
Everything works as intended locally. Great work!

**Review by jessabelledelen — APPROVED** [review]
yayayay good job :)) ❤️ promise you'll fix the logo size + the padding on the right ily
- remove darktheme toggle for navbar
- slightly enlarge sce logo to width: 70px
- profile dropdown padding left: ~46px

**Review by andrewtedi — APPROVED** [review]
wowowowowow great job tu!

**Pull request merged**